connector/apmconnector/transaction.go

```diff
@@ -375,45 +375,57 @@ func (measurement Measurement) ExclusiveTime(transaction *Transaction) int64 {
 
 func GetTransactionMetricName(span ptrace.Span) (string, TransactionType) {
 	if span.Kind() == ptrace.SpanKindConsumer || span.Kind() == ptrace.SpanKindProducer {
```

Comment:
Didn't notice this before, but I don't think span kind producer should result in a transaction.

Server is to Consumer as Client is to Producer